dvc/dagascii.py

```diff
@@ -12,6 +15,37 @@
 from grandalf.routing import EdgeViewer
 from grandalf.routing import route_with_lines
 
+from dvc.env import DVC_PAGER
+from dvc.utils import is_exec_found
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_PAGER = "less"
+DEFAULT_PAGER_FORMATTED = "{} --chop-long-lines --clear-screen".format(
+    DEFAULT_PAGER
+)
+
+
+def find_pager():
+    if not sys.stdout.isatty():
+        return pydoc.plainpager
+
+    if is_exec_found(DEFAULT_PAGER):
```

Comment:
You are looking for the default pager even if non-default one is specified, this defeats the purpose of DVC_PAGER.

Reply:
This is actually an error, thanks for catching this.
I'll fix it.

Reply:
fixed, thanks again for catching this :+1:

Reply:
should i resolved this comment?

Reply:
https://github.com/iterative/dvc/pull/2815#discussion_r354017153